kura/examples/org.eclipse.kura.raspberrypi.sensehat.example/OSGI-INF/metatype/org.eclipse.kura.raspberrypi.sensehat.example.SenseHatExample.xml

```diff
@@ -0,0 +1,125 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
```

Comment:
Is it the correct one? Only the year?
<?xml version="1.0" encoding="UTF-8"?>
<!--

    Copyright (c) 2011, 2016 Eurotech and/or its affiliates

     All rights reserved. This program and the accompanying materials
     are made available under the terms of the Eclipse Public License v1.0
     which accompanies this distribution, and is available at
     http://www.eclipse.org/legal/epl-v10.html

    Contributors:
      Eurotech

-->